### ISSUE TITLE
Documentation outdated: "g active_scaffold:resource" instead of "g active_scaffold"

### ISSUE THREAD

**Issue by ralfebert** [activescaffold/active_scaffold#520]
Readme and wiki both give the instruction to run:

```
$ bin/rails g active_scaffold User name:string
```

This doesn't work using the current master branch on Rails 5 (is it supposed to?)

```
$ bin/rails g active_scaffold User name:string
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from <class:Base> at /Users/ralf/.rvm/gems/ruby-2.3.1/bundler/gems/active_scaffold-cce0ae2d0ff3/lib/active_scaffold/extensions/action_controller_rendering.rb:17)
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from <class:LookupContext> at /Users/ralf/.rvm/gems/ruby-2.3.1/bundler/gems/active_scaffold-cce0ae2d0ff3/lib/active_scaffold/extensions/action_view_rendering.rb:8)
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from <module:RenderingHelper> at /Users/ralf/.rvm/gems/ruby-2.3.1/bundler/gems/active_scaffold-cce0ae2d0ff3/lib/active_scaffold/extensions/action_view_rendering.rb:125)
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from block in included at /Users/ralf/.rvm/gems/ruby-2.3.1/bundler/gems/active_scaffold-cce0ae2d0ff3/lib/active_scaffold/extensions/name_option_for_datetime.rb:10)
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from block in <top (required)> at /Users/ralf/.rvm/gems/ruby-2.3.1/bundler/gems/active_scaffold-cce0ae2d0ff3/lib/active_scaffold/extensions/paginator_extensions.rb:8)
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from block in <top (required)> at /Users/ralf/.rvm/gems/ruby-2.3.1/bundler/gems/active_scaffold-cce0ae2d0ff3/lib/active_scaffold/extensions/paginator_extensions.rb:26)
Running via Spring preloader in process 67594
Could not find generator 'active_scaffold'. Maybe you meant 'scss:scaffold', 'css:scaffold' or 'scaffold'
Run `rails generate --help` for more options.
```
what works is:

```
bin/rails g active_scaffold:resource User name:string
```

**Comment by scambra** [comment]
Please, try to update active_scaffold, there is something wrong with your install, cce0ae2d0ff3 commit doesn't exist

**Comment by scambra** [comment]
You should use rails g active_scaffold:resource, I fixed README instructions

**Comment by scambra** [comment]
Fixed in 9268ae5e0
